scripts/VisionSuite/blobLocator.py

```diff
@@ -11,10 +11,56 @@
 if this file is run.
 
 """
+import os, cv2
+import numpy as np
+from utils import ColorFilter
 
+def blobLocator(img,
+    visualize=False,
+    setThresholds=False,
+    bounds = {}):
 
-def blobLocator(img):
-	return (10,20),30
+    #Assume failure
+    error = True
+
+    # Manually adjust thresholding
+    if setThresholds:
+        pass
+
+    # Filter image
+    grey = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+
+    # Apply SimpleBlobDetector
+    sbd = cv2.SimpleBlobDetector()
+    blobs = sbd.detect(grey)
+
+    # TODO: Visualize
+    if visualize:
+        bgr = cv2.cvtColor(grey, cv2.COLOR_GRAY2BGR)
+        for blob in blobs:
+            cv2.circle(bgr,(int(blob.pt[0]),int(blob.pt[1])),int(blob.size/2), (0,0,255), 1)
+
+        cv2.imshow("blobDetector", bgr)
+        cv2.waitKey(0)
+
+    if error:
+        error = True
```

Comment:
This would probably been better as a pass. Currently the blob detector is unable to detect the soccer ball so it defaults returns the error flag as true. In the next pass, I hope to add something more useful to the return such as setting default x,y,size and a print for when error is true instead of all the time as is current.